Styles/product.css

```diff
@@ -0,0 +1,26 @@
+main{
+    display: grid;
+    grid-template-columns: auto 1fr;
+}
+.filter{
+    width: 18VW;
```

Comment:
why capital words here for VW ? is there any specific reason for this

Reply:
No its a typo